ssh-setpass/README.md

```diff
@@ -6,6 +6,17 @@ tagline: |
 linux: true
 ---
 
+### Files
+
+These are the files / directories that are created and/or modified with this
+install:
+
+```text
+~/.config/envman/PATH.env
+~/.local/bin/ssh-setpass
+/etc/shadow
```

Comment:
This is incorrect. It does not affect `/etc/shadow`.

It affects `~/.ssh/id_rsa`.

Reply:
You're right, fixed.